streamer/src/nonblocking/quic.rs

```diff
@@ -905,21 +910,36 @@ impl ConnectionTable {
         }
     }
 
-    fn remove_connection(&mut self, key: ConnectionTableKey, port: u16) -> bool {
+    fn remove_connection(
+        &mut self,
+        key: ConnectionTableKey,
+        port: u16,
+        maybe_stable_id: Option<usize>,
+    ) -> usize {
```

Comment:
I think we can also retain the return type and change old_size.saturating_sub(new_size) to old_size > new_size

Reply:
+1.

Ironically, the old implementation is the one where we would need to know the actual count of connections removed. With the new implementation, we should only ever remove a single connection (and thus can increment `stats.connection_removed` by 1 on true)

Reply:
I'll document it. The returned value is being used to update the stats (i.e. how many connections were actually removed). So need to return usize instead of bool.